packages/manager/modules/nasha/src/dashboard/dashboard.template.html

```diff
@@ -120,8 +120,9 @@
                     <billing-subscription-tile
                         on-error="$ctrl.alertError(error)"
                         service-infos="$ctrl.serviceInfo"
-                        user="$ctrl.user"
-                        tracking-prefix="{{ ::$ctrl.trackingPrefix }}"
+                        service-path="{{:: $ctrl.nashaApiUrl }}"
+                        with-engagement="$ctrl.isCommitmentAvailable"
+                        tracking-prefix="nasha::dashboard"
```

Comment:
Is it possible to move the prefix nasha::dashboard to constant.

Reply:
There is a task dedicated to the tracking, this will certainly be done there :+1: